standards/using-3rd-party-open-source-products.md

```diff
@@ -0,0 +1,32 @@
+# Using 3rd-party open source products
+
+In order to make effective commercial decisions, it is important to recognise the characteristics of open source software that we find desirable. Many organisations ‘open’ their code, but monetise the software through means that restrict the freedom of the user. Some open source software does not have a strong community, and can be risky to use should problems arise which the MOJ does not understand how to resolve.
+
+The following checklist must be followed, and the risks understood and considered if the criteria is not met.
+
+## A strong community exists to maintain the code
+
+If this is not present, the software presents a sustainability and security risk to the MOJ. There is no established method for assessing the maturity of open source communities, however factors to consider are:
+
+- The activity of its contributors and contributions
+- The patterns of its release history
+- The health and diversity of its community of users
+- Trust established through longevity of use
+- The ecosystem it belongs to
+
+## The responsiveness of the community to emerging security issues is acceptable, given the context of use
+
+If the open source code is used for a critical security function, the community must be very responsive to emerging vulnerabilities. Equally, a much slower response is acceptable for non-critical code libraries.  
+
+## Support for the normal use of the code can be obtained from the community at zero cost
+
+If this is not present, then this is a commercial product, and must be considered alongside other products with licensing or support costs
+If not present, then this software would be considered inferior to like-for-like FLOSS alternatives 
+
+## If commercial support is available for advanced use, a marketplace exists to choose between different suppliers
+
+If support for advanced use is only available from a single commercial provider, then this could represent a trap later in development forcing the use of this uncompetitive paid-for support
+
+## The product is fully open source. Closed source extensions or plug-ins, if they exist, aren’t required for the normal operation or scaling of the product 
```

Comment:
```markdown
### No closed source extensions or plug-ins are required for normal operation or scaling

Truly open source products don't require the purchase or licensing of
closed source extras to operate at scale. If products can only be
operated at scaled by purchasing closed-source plug-ins, this could
introduce commercial implications over time.
```

Reply:
... operated at scale by purchasing ...